crates/re_types_registry/src/components.rs

```diff
@@ -0,0 +1,509 @@
+// DO NOT EDIT! This file was auto-generated by crates/re_types_builder/src/codegen/rust/registry.rs
+use re_types_core::ComponentName;
+
+/// Information about a Rerun [`component`](re_types_core::Component), generated by codegen.
+pub struct ComponentInfo {
+    /// Markdown docstring for the component.
+    pub docstring_md: &'static str,
+}
+
+/// Runtime "reflection" about components.
+pub type ComponentRegistry = nohash_hasher::IntMap<ComponentName, ComponentInfo>;
```

Comment:
it would be nice to be able to depend on this type without depending on the registry being filled. That way I don't have to depend on _all types_. It's quite out of reach right now, but I'd really like e.g. the selection panel to not needing to depend on re_types.